sigh/lib/sigh/runner.rb

```diff
@@ -30,6 +30,7 @@ def run
           profile = create_profile!
         end
       else
+        UI.crash!("No matching provisioning pofile found and can not create a new one because you enabled `readonly`") if Sigh.config[:readonly]
```

Comment:
Please change this to `UI.user_error!` as we don't need the full stack trace here 👍

Reply:
@KrauseFx thanks for the review! its done